LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2020, Brookhaven Science Associates, Brookhaven National Laboratory
```

Comment:
This does not need to name BSA and should just name BNL, like so:

`Copyright (c) 2020, Brookhaven National Laboratory`

See @stuartcampbell's comment [on this issue](https://github.com/NSLS-II/scientific-python-cookiecutter/issues/89#issuecomment-591649337).

Reply:
OK, thanks for pointing it out. I actually copied it from SRX, I'll correct it there as well.